### PR TITLE
Add a new command to repair attached bugs that are in a wrong state

### PR DESCRIPTION
**Pull request by smunilla**

### CODE REVIEW

File: elliott [hunk 1]
```diff
@@ -700,6 +700,46 @@ Example to add standard metadata to a 3.10 images release
...
+              multiple=True,
+              default=['MODIFIED'],
+              type=click.Choice(elliottlib.constants.VALID_BUG_STATES),
+              help="Status of the bugs")
+@click.option("--to", "new_state",
+              default='VERIFIED',
+              type=click.Choice(elliottlib.constants.VALID_BUG_STATES),
+              help="Status of the bugs")
+@pass_runtime
+def repair_bugs(runtime, advisory, original_state, new_state):
```
Comment (tbielawa) [inline elliott]: Needs more docstring. Exposition and examples plz.

File: elliott [hunk 2]
```diff
@@ -700,6 +700,46 @@ Example to add standard metadata to a 3.10 images release
             code=result.status_code))
         exit(1)
 
+#
+# Repair bugs
+# advisory:repair-bugs
+#
+@cli.command("repair-bugs", short_help="Find or add MODIFED bugs to ADVISORY")
```
Comment (tbielawa) [inline elliott]: That short help may be incorrect.

File: elliott [hunk 3]
```diff
@@ -700,6 +700,46 @@ Example to add standard metadata to a 3.10 images release
...
+            click.echo("Changing BZ#{bug_id} from {initial} to {final}".format(
+                        bug_id = bug.bug_id,
+                        initial = original_state,
+                        final = new_state))
+
+            bug.setstatus(status = new_state,
+                          comment = "Elliott changed bug status from {initial} to {final}.".format(initial=original_state, final=new_state),
+                          private = True)
+
+    return
```
Comment (tbielawa) [inline elliott]: Yeah, gtfo.

File: elliott (hunk 1 above)
Comment (smunilla) [reply elliott]: ... quite. This was just to show the how easy the code was with the new library.

**Pull request merged**